src/models/pokestop.js

```diff
@@ -515,6 +517,10 @@ Pokestop.init({
         type: DataTypes.SMALLINT(5).UNSIGNED,
         defaultValue: null,
     },
+    arScanEligible: {
+        type: DataTypes.BOOLEAN,
+        defaultValue: null,
```

Comment:
```suggestion
        defaultValue: 0,
```
Since we're providing a default value of `0` probably best to keep the default as 0 always in the database.